src/pages/tutorials/index.tsx

```diff
@@ -59,10 +63,22 @@ const getStaticProps = async () => {
 		}
 	)
 
+	const crossProductSectionCollections =
+		BETTER_TOGETHER_SECTION_COLLECTION_SLUGS.map(
+			(collectionSlug: Collection['slug']) =>
+				collections.find(
+					(collection: Collection) => collection.slug === collectionSlug
+				)
+		)
+	console.log({ crossProductSectionCollections })
```

Comment:
Should we remove this before merging into assembly?

Reply:
Sure can!

Reply:
Removed in: https://github.com/hashicorp/dev-portal/pull/1977/commits/e447539d1bcfbd3acf86b27cc245c911d11875c5